assets/src/scss/layout/_navbar.scss

```diff
@@ -321,7 +329,7 @@ $navbar-default-height: 60px;
 
       &:focus,
       &:hover {
-        border: 1px dotted transparentize($white, 0.5);
+        outline: 1px dotted transparentize($white, 0.5);
```

Comment:
Is this `outline` supposed to be applied on hover too? 🤔

Reply:
Good point. I just replaced it with `outline`, but after checking this style does not result in any difference. Both with `outline` and with `border`. It's also not really needed because it's for mobile.

I'll remove this rule :broom: